tests/SearchTest.java

```diff
@@ -21,7 +21,7 @@ public void saveData() {
 
   @Test
   public void testGetData() {
-    SearchRequest searchRequest = new SearchRequest("dog", Instant.now(), "2");
+    SearchRequest searchRequest = new SearchRequest("dog", "2");
```

Comment:
It's good that you updating tests. We'll need many more of these for search, but I won't require you to write very many in this PR. A few more would be appreciated.